utils.sh

```diff
@@ -518,7 +518,7 @@ function write_pull_secret() {
     # Get a current pull secret for registry.ci.openshift.org using the token
     tmpkubeconfig=$(mktemp --tmpdir "kubeconfig--XXXXXXXXXX")
     _tmpfiles="$_tmpfiles $tmpkubeconfig"
-    oc login https://api.ci.l2s4.p1.openshiftapps.com:6443 --kubeconfig=$tmpkubeconfig --token=${CI_TOKEN}
```

Comment:
I wonder if you want to include the port in the variable.

Reply:
It didn't prove necessary to enable testing cluster-bot builds, and AFAIK this is the common port for API access, so I'd say we can add that later if it proves necessary?